mgmtsystem_review/views/mgmtsystem_review.xml

```diff
@@ -135,7 +135,7 @@
                                 </group>
                                 <newline/>
                                 <separator string="Survey Answers" colspan="4"/>
-                                <field name="response_ids" colspan="4" nolabel="1" attrs="{'readonly':[('state','=','done')]}"/>
+                                <field name="response_ids" nolabel="1" attrs="{'readonly':[('state','=','done')]}"/>
```

Comment:
@dreispt 
this is how the fields is inserted in the view.
But th Add Item Button doesn't appear I don't know why